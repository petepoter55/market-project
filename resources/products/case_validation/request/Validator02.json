{
  "username": "Scb12345",
  "startDate": "2022/04/23",
  "endDate": "2022/05/23"
}