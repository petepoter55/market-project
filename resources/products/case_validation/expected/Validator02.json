[
  "startDate: Expected format YYYY-MM-DD",
  "endDate: Expected format YYYY-MM-DD"
]